src/app/services/settings/navigation-buttons.service.ts

```diff
@@ -1,12 +1,170 @@
 let navigationButtonsServiceIsInstantiated: boolean;
 
 class NavigationButtonsService {
+	navigationButtonsId = `${PREFIX}navigation-buttons`;
+	currentFocusElt: any;
+	handlerNavigationButtons: any;
 
 	constructor() {
 		if (navigationButtonsServiceIsInstantiated) {
 			throw new Error('NavigationButtonsService is already instantiated.');
 		}
 
 		navigationButtonsServiceIsInstantiated = true;
+
+		this.handlerNavigationButtons = this.createHandlerNavigationButtons();
+	}
+
+	styleNavigationsButtons = `
+		#${this.navigationButtonsId} {
+			display: flex;
+			gap: 1rem;
+			position: fixed;
+			bottom: 1rem;
+			right: 1rem;
+			z-index: calc(infinity);
+		}
+
+		#${this.navigationButtonsId} button {
+			background: #f16e00;
+			color: #000;
+			border: none;
+			font-weight: bold;
+			padding: 1rem 2rem;
+		}
+	`;
+
+	buttonsList: string[] = ['tab', 'shiftTab', 'click', 'escape'];
+
+	setNavigationButtons = (value: string): void => {
+		this.resetNavigationButtons();
+		if (value !== DEFAULT_VALUE) {
+			stylesServiceInstance.setStyle('navigation-buttons', this.styleNavigationsButtons);
+			this.getFocusedElement();
+			this.addNavigationButtons();
+		}
+	}
+
+	resetNavigationButtons = (): void => {
+		stylesServiceInstance.removeStyle('navigation-buttons');
+		document.querySelector(`#${this.navigationButtonsId}`)?.remove();
+		document.removeEventListener('click', this.handlerNavigationButtons);
+		document.removeEventListener('focusout', this.handlerNavigationButtons);
+	}
+
+	addNavigationButtons = (): void => {
+		let fragment = document.createDocumentFragment();
+		const container = document.createElement('div');
+		container.setAttribute('id', this.navigationButtonsId);
+
+		this.buttonsList.forEach((navigationButton: string) => {
+			let btn = document.createElement('button');
+			btn.setAttribute('id', `${this.navigationButtonsId}__${navigationButton}`);
+			btn.type = 'button';
+			btn.tabIndex = -1;
+			btn.innerHTML = i18nServiceInstance.getMessage(navigationButton);
+			container.appendChild(btn);
+			fragment.appendChild(container);
+			document.body.appendChild(fragment);
+
+			let btnNav = document.querySelector(`#${this.navigationButtonsId}__${navigationButton}`);
+
+			btnNav.addEventListener('mousedown', (event) => {
+				event.preventDefault();
+				this.simulateKeyEvent(navigationButton);
+			});
+		});
+	}
+
+	simulateKeyEvent = (name: string): void => {
+		switch (name) {
+			case 'tab':
+				this.focusElement(true);
+				break;
+			case 'shiftTab':
+				this.focusElement(false);
+				break;
+			case 'click':
+				this.currentFocusElt?.click();
+				break;
+			case 'escape':
+				this.simulateKeydownEscape();
+				break;
+			default:
+				break;
+		}
+	}
+
+	/* @todo: faire un service? qui retourne tous les éléments focusable à partir de :
+	 * https://github.com/KittyGiraudel/focusable-selectors	*/
+	focusElement = (nextElement: boolean): void => {
+		const not = {
+			inert: ':not([inert]):not([inert] *)',
```

Comment:
À vérifier côté support mais tu peux utiliser une liste dans `:not()`, avec une virgule en séparateur : ça pourrait valoir le coup de simplifier tes sélecteurs ici, et wrapper le tout dans `:not()` seulement ensuite.

Qu'en dis-tu ?